subjects/gaming/stealth-boom/README.md

```diff
@@ -1,82 +1,99 @@
 ## stealth-boom
 
-In this project, you will have to create an entire stealth game using Unreal Engine.
+A stealth game.
+
+<center>
+<img src="./resources/mgsmeme.png?raw=true" style = "width: 700px !important; height: 464px !important;"/>
+</center>
 
 ### Objectives
 
-The idea of this project is to create a little 10 minutes gameplay game with missions, with stealth based gameplay and with an AI patrolling NPC.
+The goal of this project is to create a playable gameplay loop around stealth mechanics.
+
+### Scenario
+
+You are an Unreal Engine developer tasked with creating a full stealth game from scratch. You need to implement everything, including AI behavior trees, player animations, and gun mechanics. As you progress, you realize balancing these elements—ensuring the AI responds accurately to player movements while maintaining smooth gameplay—is more challenging than expected. Every feature, from the main menu to mission completion, must work seamlessly to deliver a polished and playable game within the given constraints.
+
+Good luck and remember to have fun while making the game !
+
+### Resources
```

Comment:
Resources section can be put at the end of this subject.

Reply:
done